### DISCUSSION TITLE
Matek R24-P6V sbus firmware

### DISCUSSION THREAD

**Discussion by dan-sergison** [ExpressLRS/ExpressLRS#2969] [Q&A] [answered]
I have been watching videos about how to set this rx up but the particular targets and options mentioned I cannot find.  time has moved on with newer elrs releases versions I guess.  

This is the R24-P6V and I need sbus.  I know there is a smaller cheaper sbus only rx but I need vbat telemetry and I want the vario.   I'm using sbus into a heli flybarless controller.   I see the Matek r24-D, P, S versions of r24 but no P6V and it doesn't output to sbus that my toolkitRc can see anyway.

**Comment by pkendall64** [comment]
The problem you have is that Matek is not a supported manufacturer. They have decided not to be involved with the project in any formal way. So they do not provide samples for testing and the dev team do not add targets that they cannot test and verify.

You will see on their product page they say to use the Radiomaster ER6-GV target.
![Screenshot 2024-10-05 at 10 45 37 AM](https://github.com/user-attachments/assets/1de650b0-2195-4b4e-8844-035caa507342)

I would steer clear of any Matek ELRS devices in the future.

**Reply by dan-sergison** [reply]
Thanks yes I saw that and I have exactly that one loaded now, but still not working.
Do you know of any elrs rx that can provide battery voltage telemetry?  I'm so tired of ruining batteries or falling out of the air.  not running a FC of course.

**Reply by pkendall64** [reply]
It should work by adding a wire to the VBAT pad on the RX and a common ground.
The ER6-GV from Radiomaster definitely works.

**Comment by dan-sergison** [comment]
It is sending telemetry to my tx correctly but I don't seem to be getting
any sbus signal into my heli, or into my toolkitrc sbus tester.

**Reply by pkendall64** [reply, marked as answer]
In the web UI do you have output 2 set to Serial TX? Like this screen.
![Screenshot 2024-10-05 at 1 21 53 PM](https://github.com/user-attachments/assets/04a41e14-0166-4dcd-8a7c-75e892848686)

**Reply by dan-sergison** [reply]
**THANK YOU SO MUCH for the screenshot**!   I hadn't set the serial protocol correctly.  it was set to CRSFire.  it works perfectly now.
